CHANGELOG.md

```diff
@@ -6,8 +6,9 @@
 
 #### Fixes
 
-* [#67](https://github.com/ruby-grape/grape-swagger-entity/pull/67): Various build updates - [@mscrivo](https://github.com/mscrivo).
-* Your contribution here.
+- [#67](https://github.com/ruby-grape/grape-swagger-entity/pull/67): Various build updates - [@mscrivo](https://github.com/mscrivo).
+- [#68](https://github.com/ruby-grape/grape-swagger-entity/pull/68): Properly render `example` for array exposure done using another entity - [@mscrivo](https://github.com/magni-).
```

Comment:
Should be `@magni-` in the name.

Reply:
My bad 🙈 Fixed! And Rubocop should be fixed as well.